### DISCUSSION TITLE
Convert multiple urls into PDF

### DISCUSSION THREAD

**Discussion by jaimesangcap** [gotenberg/gotenberg#1033] [Q&A] [answered]
Hi, I'm using `forms/chromium/convert/url` to generate single PDF file and it works well.

Is there a way to generate single PDF file using multiple urls? It is like merging the PDF for each URL to generate a single PDF file.

**Comment by starwalkn** [comment, marked as answer]
It seems that you need to use the downloadFrom form field. See https://github.com/gotenberg/gotenberg/releases/tag/v8.10.0

**Reply by jaimesangcap** [reply]
Thanks for sharing the link. Been trying the `downloadFrom` for hours. I did not realize I'm using an older version.
